js/src/elements/form-behavior.ts

```diff
@@ -358,7 +356,7 @@ class FormBehaviorElement extends HTMLElement {
 				continue;
 			}
 
-			if (input.hasAttribute('aria-hidden')) {
+			if (input.hasAttribute('aria-hidden') === true) {
```

Comment:
curious why this change was necessary, seems like same with but with more characters

Reply:
In an earlier PR, Ben made a comment on how `aria-hidden="false"` would trip this up. Perhaps it's an unlikely scenario to explicit set the attribute to false, but I suppose it can still happen.